libs/bixarena/api-description/src/components/schemas/VoteCreateRequest.yaml

```diff
@@ -0,0 +1,10 @@
+type: object
+description: The information used to create a new vote.
+properties:
+  battleId:
+    $ref: BattleId.yaml
```

Comment:
If the battle ID is passed as a query parameter as suggested (see below), the this property should be removed from this object.